sunpy/cm/cm.py

```diff
@@ -137,14 +143,14 @@ def get_cmap(name):
         raise ValueError("Colormap {name!s} is not recognized".format(name=name))
 
 
-def show_colormaps(filter=None):
+def show_colormaps(search=None):
```

Comment:
can you add a second changelog file under `breaking` for this please?

Reply:
So I add a file called 2731.breaking.rst ?

Reply:
yes